packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -181,6 +191,28 @@ export class ScmTreeWidget extends TreeWidget {
         return super.renderNode(node, props);
     }
 
+    protected renderIndent(node: TreeNode, depth: number): React.ReactNode {
```

Comment:
is there a way to avoid code duplication?

Reply:
@akosyakov Most of the tree use the base tree (treeWidget). I added additional css classnames in other trees (namely [scm](https://github.com/eclipse-theia/theia/blob/5d78b03a67206a335332acdfc0182dce4cd79f51/packages/scm/src/browser/scm-tree-widget.tsx#L203), [search in workspace](https://github.com/eclipse-theia/theia/blob/5d78b03a67206a335332acdfc0182dce4cd79f51/packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx#L681) and [source](https://github.com/eclipse-theia/theia/blob/5d78b03a67206a335332acdfc0182dce4cd79f51/packages/core/src/browser/source-tree/source-tree-widget.tsx#L97)) because each of those trees have different depth and specific stylings. I was not able to find a general set of stylings that would work for all trees.

Reply:
It is not good that we should copy so much code, there should be better way, like using tree props to provided a custom css class for this method. Instead of copying the entire body.